test/activemodel_validation_test.rb

```diff
@@ -282,6 +282,13 @@ def email_present?
       # merge existing errors
       form.errors.add(:policy, "another error")
       form.errors.messages.must_equal(email: ["can't be blank", "fill it out!"], username: ["not ok", "must be yo"], policy: ["error_text", "another error"])
+      # keep added errors after validate
+      form.valid?
+      form.errors.messages.must_equal(email: ["can't be blank", "fill it out!"], username: ["not ok", "must be yo"], policy: ["error_text", "another error"])
+      form.validate(username: "username", email: "email@email.com").must_equal false
+      form.errors.messages.must_equal(policy: ["error_text", "another error"], username: [], email: [])
+      form.errors.added? :policy, "error_text"
```

Comment:
what is the idea of these lines? should be `must_equal(true)`

ps: not sure if there's something `must_be_true` of something line that, I'm more used to the `assert_truthy` syntax hahah

Reply:
`ActiveModel::Errors#added?` https://apidock.com/rails/ActiveModel/Errors/added%3F it can be used as tests....testing this PR into Sam's project and it does not work!!!!!! Better `form.errors.messages` returns the correct messages but `form.errors.details` is incorrect....